### ISSUE TITLE
Numerous Divide By Zero Warnings

### ISSUE THREAD

**Issue by DarioS** [statisticalbiotechnology/diffacto#19]
I see thousands of warnings about dividing by zero. Should I, as an end-user. be concerned and do anything differently?

```
diffacto.py:171: RuntimeWarning: divide by zero encountered in double_scalars
diffacto.py:208: RuntimeWarning: invalid value encountered in true_divide
```

It also happens for the HBY20 Mix example data set. Sometimes, `ss_resid` is 0. Please document this phenomenon for users to know more about it. I also frequently see `-inf` in the S/N column. Should the columns of the results table also be documented?

**Comment by percolator** [comment]
Hi Dario,

I think this relates to some type of missing values. Would you be able to
share data, so that we can recreate the error.

Yours
--Lukas

**Comment by DarioS** [comment]
It can be triggered by running Example-2 (HBY20) in the examples directory of this repository using the example command shown.

**Comment by rcpoulos** [comment]
I would also like to query the -inf values often returned in the S/N column. Should these proteins be treated as below the S/N threshold (eg -20) and therefore be discarded?

**Comment by percolator** [comment]
Yes, proteins with S/N of -inf are indeed below any threshold and should be
treated with caution. --Lukas
